base/client.js

```diff
@@ -97,7 +95,7 @@ class BugsnagClient {
   }
 
   leaveBreadcrumb (name, metaData, type, timestamp) {
-    if (!this._configured) throw new Error('Bugsnag must be configured before calling leaveBreadcrumb()')
+    if (!this._configured) throw new Error('client not configured')
```

Comment:
If I understand things correctly, this change isn't necessarily required for the on-by-default session tracking? The change itself is fine, but might be better in a separate PR in future to make review easier

Reply:
It's in a separate commit with its own message.

The functionality isn't related to the session tracking change, but the changes made the bundle size increase, so this change is required to land at the same time to keep the bundle size reasonable.